docs/modules/overview/getting-started.md

```diff
@@ -30,4 +32,4 @@ The template will attempt to auto-fill details in your manifest as well as defin
 3. Ensure the Blish HUD and all of Blish HUD's dependent references have **Copy Local** set to **False** in the properties of the reference.
    - If you don't do this, your module will package these assemblies with your module.
      - This will make your modules unnecessarily large (it'll contain the entire Blish HUD assembly!).
-     - Blish HUD will already have all of these assemblies loaded (since it naturally depends on them), so you don't need to include them.
\ No newline at end of file
+     - Blish HUD will already have all of these assemblies loaded (since it naturally depends on them), so you don't need to include them.
```

Comment:
Not sure why it highlights this on the github editor. Maybe there's a whitespace change?